tools/scripts/data.toml

```diff
@@ -183,9 +183,3 @@ args = [
     "--all-locales",
     "--ignore-missing-data",
 ]
-
-[tasks.icu4x-key-extract]
```

Comment:
Issue: Please restore the CI job that tests this.

Reply:
This is now in a unit test so it's in `ci-job-test`.